readme.md

```diff
@@ -1,151 +1,125 @@
-- Open source (installable via npm/Yarn or Docker Hub) so can be self-hosted or used via the Mend Renovate App
+# What is Mend Renovate CLI?
 
-## Supported Platforms
+Renovate is an automated dependency update tool.
+It helps to update dependencies in your code without needing to do it manually.
+When Renovate runs on your repo, it looks for references to dependencies (both public and private) and, if there are newer versions available, Renovate can create pull requests to update your versions automatically.
 
-Renovate works on these platforms:
+## Features
 
-- [GitHub (.com and Enterprise Server)](https://docs.renovatebot.com/modules/platform/github/)
-- [GitLab (.com and CE/EE)](https://docs.renovatebot.com/modules/platform/gitlab/)
-- [Bitbucket Cloud](https://docs.renovatebot.com/modules/platform/bitbucket/)
-- [Bitbucket Server](https://docs.renovatebot.com/modules/platform/bitbucket-server/)
-- [Azure DevOps](https://docs.renovatebot.com/modules/platform/azure/)
-- [AWS CodeCommit](https://docs.renovatebot.com/modules/platform/codecommit/)
-- [Gitea and Forgejo](https://docs.renovatebot.com/modules/platform/gitea/)
-- [Gerrit (experimental)](https://docs.renovatebot.com/modules/platform/gerrit/)
+- Delivers update PRs directly to your repo
+  - Relevant package files are discovered automatically
+  - Pull Requests automatically generated in your repo
+- Provides useful information to help you decide which updates to accept (age, adoption, pass rates, merge confidence)
+- Highly configurable and flexible to fit in with your needs and repository standards
+- Largest collection of languages and platforms (listed below)
+- Connects with private repositories and package registries
 
-## Who Uses Renovate?
+**Languages**
 
-Renovate is widely used in the developer community:
+Renovate can provide updates for nearly all languages, platforms, and registries including: npm, Java, Python, .NET, Scala, Ruby, Go, Docker and more.
+Supports over 90 different package managers.
 
-![Logos of companies and projects that use Renovate](./docs/usage/assets/images/matrix.png)
+**Platforms**
 
-<details>
-<summary>List of companies and projects that use Renovate</summary>
+Renovate updates code repositories on the following platforms: GitHub, GitLab, Bitbucket, Azure DevOps, AWS Code Commit, Gitea, Forgejo, Gerrit (experimental)
 
-- Prisma
-- Netlify
-- Envoy
-- Condé Nast
-- Microsoft
-- Atlassian
-- Sourcegraph
-- Mozilla
-- Deloitte
-- Telus
-- Yarn
-- HashiCorp
-- Automattic
-- Algolia
-- eBay
-- Cypress
-- Red Hat
-- Financial Times
-- Uber
-- Buildkite
+#  Ways to Run Renovate
 
-</details>
+The most effective way to run Renovate is to use an automated job scheduling system that regularly runs Renovate on all enabled repositories and responds with priority to user activity. Mend offers cloud-hosted and self-hosted solutions. See the options below.
```

Comment:
Please make sure to use one sentence per line for the entire document (this is the markdown style guide recommendation for this repo)